packages/core/directives/polyline.ts

```diff
@@ -260,10 +260,3 @@ export class AgmPolyline implements OnDestroy, OnChanges, AfterContentInit {
     this._subscriptions.forEach((s) => s.unsubscribe());
   }
 }
-
-export interface PathEvent {
```

Comment:
Because this is an interface TypeScript will allow you to use either of these and will not tell you anything about them being different and that's why I removed them. If they are **classes** they are actually generated in the output and you can tell one from another and it's more important to have them called out.

Defining them as interface doesn't necessarily _hurt_, but it just depends _why_ you are adding extra interfaces.

Reply:
I am adding extra interfaces, because this specific interface is simplified for the user, and hides implementation details